1-JavaScript/Week4/homework/ex5-wallet/index.js

```diff
@@ -113,6 +113,6 @@ const quiz = {
       b: 'Joe', 
       c: 'Jane' 
     },
-    answer: undefined,
+    answer: C,
```

Comment:
@Mehran-Banbersta Great job but the answer should be String ' ' 

```suggestion
    answer: 'c',
```

Reply:
@Mehran-Banbersta Missing q4